Backend/src/views/mails/invitation_to_team/text.txt

```diff
@@ -0,0 +1,8 @@
+Invitation to a team
+
+Hey,
+We want to inform you that you have invited {{firstname}} to the team {{teamname}}.
```

Comment:
The text of the .txt file is completely different to the .html file

We wanted to inform you that you were invited to the team {{teamname}}
vs
We want to inform you that you have invited {{firstname}} to the team {{teamname}}

Reply:
In this case you need to add another html file and another txt file, because every email has to be available as txt as well as html